### PR TITLE
fix: remove unnecessary quotation marks

### PR DESCRIPTION
**Pull request by narinluangrath**
one liner

### CODE REVIEW

**Review by audwinoyong — APPROVED** [review]

**Comment by robertguss** [comment]
@narinluangrath can you please sign the CLA agreement for this repo? I will be happy to merge this once you do. We cannot merge until that agreement is signed. Thanks. 

https://cla-assistant.io/cypress-io/cypress-documentation?pullRequest=4301

**Comment by narinluangrath** [comment]
@robertguss Done! Sorry for the delay.

**Review by robertguss — APPROVED** [review]

**Comment by robertguss** [comment]
@narinluangrath Thank you 👍

**Pull request merged**
